packages/eslint-config-airbnb-base/README.md

```diff
@@ -0,0 +1,33 @@
+# eslint-config-airbnb-base
+
+[![npm version](https://badge.fury.io/js/eslint-config-airbnb-base.svg)](http://badge.fury.io/js/eslint-config-airbnb-base)
+
+This package provides Airbnb's base JS .eslintrc as an extensible shared config.
+
+## Usage
+
+We export two ESLint configurations for your usage.
+
+### eslint-config-airbnb-base
+
+Our default export contains all of our ESLint rules, including ECMAScript 6+ and React. It requires `eslint` and `eslint-plugin-import`.
+
+1. `npm install --save-dev eslint-config-airbnb-base eslint-plugin-import eslint`
+2. add `"extends": "airbnb-base"` to your .eslintrc
+
+### eslint-config-airbnb-base/legacy
+
+Lints ES5 and below. Requires `eslint` and `eslint-plugin-import`.
+
+1. `npm install --save-dev eslint-config-airbnb eslint-plugin-import eslint`
+2. add `"extends": "airbnb/legacy"` to your .eslintrc
```

Comment:
I think you might still have a few things that need to be changed to `airbnb-base` in this file.